o1vm/src/pickles/prover.rs

```diff
@@ -176,7 +184,48 @@ where
         }
     };
 
-    // FIXME: add quotient polynomial
+    let quotient_poly: DensePolynomial<G::ScalarField> = {
+        // Compute ∑ α^i constraint_i as an expression
+        let combined_expr =
+            E::combine_constraints(0..(constraints.len() as u32), (constraints).to_vec());
```

Comment:
not a big deal, but I'd rather have this computed elsewhere
The expressions should not be computed at proving time, for separation of concept